src/js/bootstrap-slider.js

```diff
@@ -1050,6 +1064,28 @@ const windowIsDefined = (typeof window === "object");
 				this.handle2.style[this.stylePos] = positionPercentages[1]+'%';
 				this.handle2.setAttribute('aria-valuenow', this._state.value[1]);
 
+				/* Position highlight range elements */
+				if (this.rangeHighlightElements.length > 0 && Array.isArray(this.options.rangeHighlights) && this.options.rangeHighlights.length > 0) {
+					for (i = 0; i < this.options.rangeHighlights.length; i++) {
+						var startPercent = this._toPercentage(this.options.rangeHighlights[i].start);
+						var endPercent = this._toPercentage(this.options.rangeHighlights[i].end);
+
+						var currentRange = this._createHighlightRange(startPercent, endPercent);
+
+						if (currentRange) {
+							if (this.options.orientation === 'vertical') {
+								this.rangeHighlightElements[i].style.top = currentRange.start + '%';
```

Comment:
could you write these using ES6 template string format instead of concatenation?

```
this.rangeHighlightElements[i].style.top = `${currentRange.start}%`;
```